packages/core/components/EditMetadata/MetadataDetails.tsx

```diff
@@ -41,20 +52,95 @@ export default function EditMetadataDetailsList(props: DetailsListProps) {
         }
         return <></>;
     };
+    const annotationFormatter = annotationFormatterFactory(
+        props.fieldType || AnnotationType.STRING
+    );
 
     function renderItemColumn(
         item: ValueCountItem,
         _: number | undefined,
         column: IColumn | undefined
     ) {
         const fieldContent = item[column?.fieldName as keyof ValueCountItem] as string;
-        if (!fieldContent) return "[No value] (blank)";
-        if (column?.fieldName === "fileCount") {
-            return <div className={styles.columnRightAlignCell}>{fieldContent}</div>;
+        if (column?.fieldName === "value") {
+            if (!fieldContent) return "[No value] (blank)";
+            else return annotationFormatter.displayValue(fieldContent);
+        } else if (column?.fieldName === "fileCount") {
+            return <div className={styles.columnRightAlignCell}>{fieldContent || 0}</div>;
         }
         return fieldContent;
     }
 
+    const inputField = () => {
+        switch (props.fieldType) {
+            case AnnotationType.DATE:
+            case AnnotationType.DATETIME:
```

Comment:
Is there no time selector for `DATETIME` types or is there always a time selector for `DATE` types? Either of those would be bad no?

Reply:
Unfortunately yes, the date picker does not provide a time selector. This issue also exists in the AnnotationFilterForm

Reply:
I'm opening a separate ticket for this because the FluentUI TimePicker is imo too buggy to use (and the API does not match the component), so making our own is just as good of an option but will hold up this ticket

Reply:
added in [9328a44](https://github.com/AllenInstitute/biofile-finder/pull/354/commits/9328a44ecfedb6b82b1fedfbbaf73679dbb37633)
<img width="500" alt="image" src="https://github.com/user-attachments/assets/85556131-ee0e-43ad-a061-3fdda2641d9f" />